README.md

```diff
@@ -27,7 +27,7 @@ The python-xmlsec package on Windows is having some issues with hosting, so you
 `pip install
 https://github.com/mehcode/python-xmlsec/releases/download/1.3.5/xmlsec-1.3.52.dev0-cp36-cp36m-win_amd64.whl`
 
-For more information [here is the related issue](https://github.com/onelogin/python3-saml/issues/110). 
```

Comment:
ah, apologies, I have my editor set to remove trailing spaces. I'll revert that part.